src/transcription/live.rs

```diff
@@ -2,7 +2,11 @@
 // Currently not documented because interface of this module is still changing
 #![allow(missing_docs)]
 
-//! Provides various types that are used for streaming transcription requests to Deepgram.
+//! Types used for live audio transcription.
+//!
+//! See the [Deepgram API Reference][api] for more info.
+//!
+//! [api]: https://developers.deepgram.com/api-reference/#transcription-streaming
```

Comment:
If the documentation calls this mode "streaming" wouldn't it make sense to follow that same terminology here?  I don't see us using the term "live" other places.

Reply:
I have been basing all naming so far off of the Python SDK. I do think that "streaming" makes more sense, though.